lib/rules/selector-no-vendor-prefix/__tests__/index.js

```diff
@@ -87,3 +87,27 @@ testRule(rule, {
     }
   ]
 });
+
+testRule(rule, {
+  ruleName,
+  config: [
+    true,
+    { ignoreProperties: ["::-webkit-input-placeholder", "/-moz-.*/"] }
+  ],
+
+  accept: [
+    {
+      code: "input::-webkit-input-placeholder { color: pink; }"
+    },
+    {
+      code: "input::-moz-placeholder { color: pink; }"
+    }
+  ],
+
+  reject: [
+    {
+      code: "input:-ms-input-placeholder { color: pink; }",
```

Comment:
I think this might need to a pseudo-element to appease appveyor: `input::-ms-input-placeholder`.